rules/integrations/aws/persistence_redshift_instance_creation.toml

```
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2022/04/12"
+maturity = "production"
+updated_date = "2022/04/12"
+integration = "aws"
+
+[rule]
+author = ["Elastic", "Pete Hampton"]
```

```suggestion
author = ["Elastic"]
```
We usually only include authors other than Elastic in community contributions